app/views/gobierto_admin/gobierto_dashboards/dashboards/list.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="modal">
+  <h2>Lorem ipsum dolor sit amet consectetur adipisicing elit. Blanditiis, illo.</h2>
```

Comment:
Eliminatem latim

Reply:
@entantoencuanto this file should be removed, shouldn't it?

Reply:
Yes, I´ll fix it

Reply:
No no wait

Reply:
I was mixing stuff, this is another PR. Do not pay attention to this file @ferblape. I guess this gonna be removed when master is merged

Reply:
Resolve this comments as they're gonna be replaced in the consequent PRs